app/views/claims/student_loan_amount.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render("shared/error_summary", instance: current_claim) if current_claim.errors.any? %>
+
+    <%= form_for current_claim, url: claim_path do |form| %>
+      <h1 class="govuk-heading-xl">
+        How much student loan did you repay while you were at <%= current_claim.claim_school_name %> between 6 April XXXX and 5 April XXXX?
+      </h1>
+
+      <%= form.label :student_loan_repayment_amount, t("tslr.questions.student_loan_amount"), class: "govuk-label"  %>
```

Comment:
I feel like this label is possibly redundant, and the question in the heading above should be the actual form label. Also making the question the label would make this page and the way it is played back on the confirmation page more consistent with the rest of the questions.

Reply:
@tekin As discussed I've made the question the label and used it on the "check your answers" page. Ready for another look